webpack.config.js

```diff
@@ -2,14 +2,21 @@ const path = require('path');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const { CleanWebpackPlugin } = require('clean-webpack-plugin');
 
-const genConfig = (filename, { inlineCss } = {}) => ({
+const genConfig = (outputFilename, { inlineCss, inputFilename = 'index.js' } = {}) => ({
   output: {
     path: path.resolve(__dirname, 'dist'),
-    filename,
+    filename: outputFilename,
     library: 'superheroUtils',
     libraryTarget: 'umd'
   },
+  entry: {
+    entry: `./src/${inputFilename}`,
+  },
```

Comment:
Let's try one more time:
```suggestion
  entry: `./src/${inputFilename}`,
```

Reply:
thanks